source/includes/destinationDataHandling-introduction.rst

```diff
@@ -0,0 +1,4 @@
+Starting in ``mongosync-beta`` 1.8, use the ``destinationDataHandling``
```

Comment:
We've switched to using "`mongosync` beta 1.8", see [DOCSP-40404](https://jira.mongodb.org/browse/DOCSP-40404) for more details.